app/views/course/experience_points_records/_experience_points_record.html.slim

```diff
@@ -1,14 +1,32 @@
-= content_tag_for(:tr, experience_points_record)
-  td = link_to_user(@course_user_preload_service.course_user_for(experience_points_record.updater))
-  td
-    - if experience_points_record.manually_awarded?
-      = format_inline_text(experience_points_record.reason)
-    - else
-      = render partial: experience_points_record.specific, suffix: 'experience_points_reason'
-  td = experience_points_record.points_awarded
-  td = experience_points_record.updated_at
-  td
-    - if experience_points_record.manually_awarded? && can?(:destroy, experience_points_record)
-      = delete_button(course_user_experience_points_record_path(current_course,
-                                                                   @course_user,
-                                                                   experience_points_record))
+- can_update = can?(:update, experience_points_record)
+- points_record_path = course_user_experience_points_record_path(current_course,
+                                                                 @course_user,
+                                                                 experience_points_record)
+- row_options_hash = { class: 'experience-points-record' }.tap do |hash|
+  - hash.merge!({ 'data-action' => points_record_path, 'data-method' => 'patch' }) if can_update
+
+= content_tag_for(:tr, experience_points_record, row_options_hash)
+  = simple_fields_for experience_points_record do |f|
```

Comment:
Do you think it might be better to split the views into `course_students vs course_staff`, or `manually_awarded vs automatically awarded`? Or if you feel that this is the best method, I think that's all right too. I'm just seeing some repeated checks here.